.github/workflows/master-e2e.yaml

```diff
@@ -398,14 +398,20 @@ jobs:
       - name: Configure Rancher & Libvirt
         if: inputs.test_type == 'cli'
         run: cd tests && make e2e-configure-rancher
-      - name: Bootstrap node 1, 2 and 3 (with iPXE) in pool "master" (use Emulated TPM if possible)
+      - name: Bootstrap node 1, 2 and 3 in pool "master" (use Emulated TPM if possible)
         if: inputs.test_type == 'cli'
         env:
           EMULATE_TPM: true
           POOL: master
           VM_START: 1
           VM_END: 3
         run: |
+          # Only use ISO boot if the upstream cluster is RKE2
+          # due to issue with pxe, dhcp traffic
+          if ${{ contains(inputs.upstream_cluster_version, 'rke') }}; then
+            export ISO_BOOT=true
+          fi
+
```

Comment:
Can be removed too (sorry I forgot this one in the last comment...). I removed most of this kind of lines previously, to try to keep the file "smaller" ;-)